SIGSEGV: segmentation violation running repair with mgmt 2.5.3 deployed by operator

Scylla manager version: 2.5.3 (docker)
Scylla Operator version: v1.7.2
Scylla version: 2022.1.rc3

Description:
If we run `repair` command then we get following error in manager logs:

```
{"L":"INFO","T":"2022-05-06T15:29:05.574Z","N":"scheduler","M":"Task started","cluster_id":"e68ecf69-abe8-49a3-afef-340b41999c8c","task_type":"repair","task_id":"2f1bb970-fe17-48ac-8aee-96598deb78c9","run_id":"31f48d70-cd51-11ec-a757-6e1531e4f5ed","_trace_id":"n1ut07bOSlWwbieeSTh6zw"}
{"L":"INFO","T":"2022-05-06T15:29:05.699Z","N":"repair","M":"Repair of the following keyspaces will be skipped because not all the tokens are present in the specified DCs","keyspaces":"audit","_trace_id":"n1ut07bOSlWwbieeSTh6zw"}
{"L":"INFO","T":"2022-05-06T15:29:05.701Z","N":"repair","M":"Initializing repair","cluster_id":"e68ecf69-abe8-49a3-afef-340b41999c8c","task_id":"2f1bb970-fe17-48ac-8aee-96598deb78c9","run_id":"31f48d70-cd51-11ec-a757-6e1531e4f5ed","target":{"units":[{"Keyspace":"system_auth","Tables":["role_attributes","role_members","roles"],"AllTables":true},{"Keyspace":"system_distributed","Tables":["service_levels","cdc_streams_descriptions_v2","cdc_generation_timestamps","view_build_status"],"AllTables":true},{"Keyspace":"system_distributed_everywhere","Tables":["cdc_generation_descriptions_v2"],"AllTables":true},{"Keyspace":"system_traces","Tables":["events","node_slow_log","sessions","sessions_time_idx","node_slow_log_time_idx"],"AllTables":true}],"dc":["dc-1"],"fail_fast":false,"continue":true,"intensity":1,"parallel":0,"small_table_threshold":1073741824},"_trace_id":"n1ut07bOSlWwbieeSTh6zw"}
{"L":"INFO","T":"2022-05-06T15:29:06.317Z","N":"repair","M":"Detected small tables","tables":["system_auth.role_attributes","system_auth.role_members","system_auth.roles","system_distributed.service_levels","system_distributed.cdc_streams_descriptions_v2","system_distributed.cdc_generation_timestamps","system_distributed.view_build_status","system_distributed_everywhere.cdc_generation_descriptions_v2","system_traces.events","system_traces.node_slow_log","system_traces.sessions","system_traces.sessions_time_idx","system_traces.node_slow_log_time_idx"],"threshold":1073741824,"_trace_id":"n1ut07bOSlWwbieeSTh6zw"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x12a8bb7]

goroutine 1635 [running]:
github.com/scylladb/scylla-manager/pkg/scyllaclient.(*Client).TotalMemory(0xc0006d7040, 0x2d4a128, 0xc000269840, 0xc000928020, 0xc, 0x0, 0x405cc5, 0xc000585020)
	github.com/scylladb/scylla-manager/pkg/scyllaclient/client_scylla.go:762 +0x117
github.com/scylladb/scylla-manager/pkg/service/repair.(*Service).hostRangeLimits.func1(0x2, 0xc00061d800, 0x0)
	github.com/scylladb/scylla-manager/pkg/service/repair/service.go:506 +0xd0
github.com/scylladb/scylla-manager/pkg/util/parallel.Run.func1(0xc0005b0460, 0x3, 0xc0005b0468, 0xc0005790e0, 0xc000a854f0)
	github.com/scylladb/scylla-manager/pkg/util/parallel/parallel.go:69 +0x7f
created by github.com/scylladb/scylla-manager/pkg/util/parallel.Run
	github.com/scylladb/scylla-manager/pkg/util/parallel/parallel.go:54 +0x132
```

The same happens also using dockerized scylla manager `2.5.2`

--- a/pkg/scyllaclient/client_scylla.go
+++ b/pkg/scyllaclient/client_scylla.go
@@ -761,9 +761,9 @@ func (c *Client) TotalMemory(ctx context.Context, host string) (int64, error) {
 	var totalMemory int64
 	for _, m := range metrics[metricName].Metric {
 		switch {
-		case m.Counter != nil:
+		case m.Counter != nil && m.Counter.Value != nil:
 			totalMemory += int64(*m.Counter.Value)
-		case m.Gauge != nil:
+		case m.Gauge != nil && m.Gauge.Value != nil:
 			totalMemory += int64(*m.Gauge.Value)
 		}
 	}

